preferences/Preference.js

```diff
@@ -1,19 +1,23 @@
 // See https://github.com/jenkinsci/js-storage
-const storage = require('@jenkins-cd/storage');
-const jenkinsNS = storage.jenkinsNamespace();
-const preferences = jenkinsNS.subspace('preferences');
+const storage = require('@jenkins-cd/storage/storage');
+const StorageNamespace = require('@jenkins-cd/storage/StorageNamespace');
```

Comment:
Hi Keith, I guess you are talking about the js-bundler and say this out of experience. The above is a pretty common pattern in es5 to access a sub module. Sometimes it is even a design pattern to not include certain files in the standard exports of the module to not init them by solely importing them in another file. 

In the js-storage that is not the case and I can create a PR to export them, but I tend to say we need to fix the js-bundler if your statement is correct, since otherwise we will get much more phantom bugs from npm that no-one can logically explain. ...I guess you are familiar with those ;)